app/expertise/page.tsx

```
@@ -1,54 +1,38 @@
+import React from "react";
+import { Card, CardContent, CardDescription, CardFooter, CardHeader, CardTitle, Badge } from "@/components/ui";
+import { ExpertiseList } from "@/data/expertise-list";
 
 export const metadata = {
-	title: 'Expertise - Froilan | Web Developer',
-	description: 'Personal Website of Froilan',
+    title: "Expertise | Froilan J. Aquino | Web Developer",
+    description: "Personal Website of Froilan",
 };
 
 const Expertise = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)